tests/Unit/Domain/CoordinateMaps/Test_CubicScale.cpp

```diff
@@ -0,0 +1,281 @@
+        scale_map_deserialized.inverse(mapped_point, t, f_of_t_list), point_xi);
+
+    const double jacobian =
+        a + 3.0 * (b - a) * square(point_xi[0] / outer_boundary);
+    const double inv_jacobian =
+        1.0 / (a + 3.0 * (b - a) * square(point_xi[0] / outer_boundary));
+
+    CHECK(scale_map.jacobian(point_xi, t, f_of_t_list).get(0, 0) == jacobian);
+    CHECK(scale_map.inv_jacobian(point_xi, t, f_of_t_list).get(0, 0) ==
+          inv_jacobian);
+    CHECK(scale_map_deserialized.jacobian(point_xi, t, f_of_t_list).get(0, 0) ==
+          jacobian);
+    CHECK(scale_map_deserialized.inv_jacobian(point_xi, t, f_of_t_list)
+              .get(0, 0) == inv_jacobian);
+
+    const double a_dot = f_of_t_a.func_and_deriv(t)[1][0];
+    const double b_dot = f_of_t_b.func_and_deriv(t)[1][0];
+    const std::array<double, 1> frame_vel =
+        point_xi *
+        (a_dot + (b_dot - a_dot) * square(point_xi[0] / outer_boundary));
+
+    CHECK_ITERABLE_APPROX(scale_map.frame_velocity(point_xi, t, f_of_t_list),
+                          frame_vel);
+    CHECK_ITERABLE_APPROX(
+        scale_map_deserialized.frame_velocity(point_xi, t, f_of_t_list),
+        frame_vel);
+
+    const double a_ddot = f_of_t_a.func_and_2_derivs(t)[2][0];
+    const double b_ddot = f_of_t_b.func_and_2_derivs(t)[2][0];
+    const double time_time =
+        a_ddot * point_xi[0] +
+        (b_ddot - a_ddot) * cube(point_xi[0]) / square(outer_boundary);
+    const double time_space =
+        a_dot + 3.0 * (b_dot - a_dot) * square(point_xi[0] / outer_boundary);
+    const double space_space =
+        6.0 * (b - a) * point_xi[0] / square(outer_boundary);
+
+    CHECK(scale_map.hessian(point_xi, t, f_of_t_list).get(0, 0, 0) ==
+          time_time);
+    CHECK(scale_map.hessian(point_xi, t, f_of_t_list).get(0, 1, 0) ==
+          time_space);
+    CHECK(scale_map.hessian(point_xi, t, f_of_t_list).get(0, 0, 1) ==
+          time_space);
+    CHECK(scale_map.hessian(point_xi, t, f_of_t_list).get(0, 1, 1) ==
+          space_space);
+
+    CHECK(
+        scale_map_deserialized.hessian(point_xi, t, f_of_t_list).get(0, 0, 0) ==
+        time_time);
+    CHECK(
+        scale_map_deserialized.hessian(point_xi, t, f_of_t_list).get(0, 1, 0) ==
+        time_space);
+    CHECK(
+        scale_map_deserialized.hessian(point_xi, t, f_of_t_list).get(0, 0, 1) ==
+        time_space);
+    CHECK(
+        scale_map_deserialized.hessian(point_xi, t, f_of_t_list).get(0, 1, 1) ==
+        space_space);
+
+    // Check inequivalence operator
+    CHECK_FALSE(scale_map != scale_map);
+    CHECK_FALSE(scale_map_deserialized != scale_map_deserialized);
+
+    // Check serialization
+    CHECK(scale_map == scale_map_deserialized);
+    CHECK_FALSE(scale_map != scale_map_deserialized);
+
+    test_coordinate_map_argument_types(scale_map, point_xi, t, f_of_t_list);
+
+    t += dt;
+  }
+}
+
+SPECTRE_TEST_CASE("Unit.Domain.CoordMapsTimeDependent.CubicScale.MultipleRoots",
```

Comment:
you can combine the common parts of the interior of these tests into a single function